docs/presentation/Customization/SortingRule.md

```diff
@@ -52,23 +130,73 @@ other non ECInstance type of nodes.
+| [`priority`](#attribute-priority-1)               | No        | `number`                                                             | `1000`                         |
+| [`condition`](#attribute-condition-1)             | No        | [ECExpression](../hierarchies/ECExpressions.md#rule-condition)       | `""`                           |
+| [`class`](#attribute-class-1)                     | No        | `SingleSchemaClassSpecification`                                     | All classes in current context |
+| [`isPolymorphic`](#attribute-ispolymorphic-1)     | No        | `boolean`                                                            | false                          |
+
+### Attribute: `requiredSchemas`
+
+> **Default value:** `[]`
+
+A list of ECSchema requirements that need to be met for the rule to be used. See more details in [Defining ECSchema Requirements for Presentation Rules](../Advanced/SchemaRequirements.md).
+
+```ts
+[[include:Sorting.RequiredSchemas.Ruleset]]
 ```
+
+### Attribute: `priority`
+
+> **Default value:** `1000`
+
+Defines the order in which rules are handled - higher priority means the rule is handled first. If priorities are equal, the rules are handled in the order they're defined.
+
+```ts
+[[include:DisabledSortingRule.Priority.Ruleset]]
+```
+
+![Example of using priority attribute](./media/sortingrule-unsorted.png)
+
+### Attribute: `condition`
+
+> **Default value:** `""`
+
+Defines a condition which needs to be met in order for the rule to be used. The condition is an [ECExpression](./ECExpressions.md#rule-condition) which has to evaluate to a boolean value.
+
+```ts
+[[include:SortingRule.Condition.Ruleset]]
+```
+
+| Ruleset variable values   | Result                                                                 |
+| ------------------------- | ---------------------------------------------------------------------- |
+| `SORT_INSTANCES = true`   | ![Example of using rule condition](./media/sortingrule-sorted-asc.png) |
+| `SORT_INSTANCES = false`  | ![Example of using rule condition](./media/sortingrule-unsorted.png)   |
+
+### Attribute: `class`
+
+Specifies ECClass whose instances should not be sorted.
+
+```ts
+[[include:SortingRule.Class.Ruleset]]
+```
+
+![Example of using class attribute](./media/sortingrule-sorted-asc.png)
+
+### Attribute: `isPolymorphic`
+
+> **Default value:** `false`
+
+Specifies that `class` attribute defined in this rule should be handled polymorphically.
+
+```ts
+[[include:SortingRule.IsPolymorphic.Ruleset]]
+```
+
+| `isPolymorphic` values    | Result                                                                          |
+| ------------------------- | ------------------------------------------------------------------------------- |
+| `isPolymorphic = true`    | ![Example of using isPolymorphic attribute](./media/sortingrule-sorted-asc.png) |
+| `isPolymorphic = false`   | ![Example of using isPolymorphic attribute](./media/sortingrule-unsorted.png)   |
```

Comment:
Can you select classes with sorting rule and disabled sorting rule such that only a subset of instances get sorted at the same hierarchy level?

Reply:
Added better examples for DisabledSorting rule